exchanges/order/order_test.go

```diff
@@ -267,6 +268,27 @@ func TestFilterOrdersByType(t *testing.T) {
 	}
 }
 
+var filterOrdersByTypeBenchmark = &[]Detail{
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+	{Type: Limit},
+}
+
+//  392455	      3226 ns/op	   15840 B/op	       5 allocs/op // PREV
```

Comment:
That's some nice savings 😎

Reply:
![image](https://user-images.githubusercontent.com/9391715/166169708-11491243-a1f7-468c-9af9-98b92d597106.png)